### PR TITLE
TPD-1515 TPD-1534 TPD-1542 TPD-1533 TPD-1592 More Ticketing Improvements

### PR DESCRIPTION
**Pull request by davidcarroll**
Handle BackButton on EasyLogin page to show message about tickets being released then proceeding to do the release and go back to the Seating chart for the event being worked on.
Also Implement Quick Search for Assisted Booking
[touchpointsoftware.atlassian.net/browse/TPD-1515](https://touchpointsoftware.atlassian.net/browse/TPD-1515)
[touchpointsoftware.atlassian.net/browse/TPD-1534](https://touchpointsoftware.atlassian.net/browse/TPD-1534)
[touchpointsoftware.atlassian.net/browse/TPD-1542](https://touchpointsoftware.atlassian.net/browse/TPD-1542)
[touchpointsoftware.atlassian.net/browse/TPD-1533](https://touchpointsoftware.atlassian.net/browse/TPD-1533)
[touchpointsoftware.atlassian.net/browse/TPD-1592](https://touchpointsoftware.atlassian.net/browse/TPD-1592)

### CODE REVIEW

File: CmsData/CmsData.csproj [hunk 1]
```diff
@@ -215,8 +215,8 @@
     <Reference Include="RestSharp, Version=106.5.4.0, Culture=neutral, PublicKeyToken=598062e77f915f75, processorArchitecture=MSIL">
       <HintPath>..\packages\RestSharp.106.5.4\lib\net452\RestSharp.dll</HintPath>
     </Reference>
-    <Reference Include="SeatsioDotNet, Version=70.9.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\SeatsioDotNet.70.9.0\lib\net452\SeatsioDotNet.dll</HintPath>
+    <Reference Include="SeatsioDotNet, Version=70.13.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\SeatsioDotNet.70.13.0\lib\net452\SeatsioDotNet.dll</HintPath>
```
Comment (brondavies) [inline CmsData/CmsData.csproj:219]: do you think we can update RestSharp now?  This is what's causing that issue with docusign.

Comment (brondavies) [reply CmsData/CmsData.csproj:219]: Actually, they have version 73 out now.

Comment (davidcarroll) [reply CmsData/CmsData.csproj:219]: I'll give it a try tomorrow with my local machine

File: CmsWeb/Areas/OnlineReg/Models/OnlineReg/AssistedSearch.cs [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+namespace CmsWeb.Areas.OnlineReg.Models
+{
+    public class AssistedSearch
+    {
+        private CMSDataContext CurrentDatabase { get; }
+        public AssistedSearch(CMSDataContext db)
+        {
+            CurrentDatabase = db;
+        }
+        public IEnumerable<SearchInfo> Search(string q, int limit)
```
Comment (brondavies) [inline CmsWeb/Areas/OnlineReg/Models/OnlineReg/AssistedSearch.cs:18]: This should probably be merged with `PeopleUtils.GetPeopleList` but we can refactor that later.

File: CmsData/CmsData.csproj (hunk 1 above)
Comment (davidcarroll) [reply CmsData/CmsData.csproj]: > do you think we can update RestSharp now? This is what's causing that issue with docusign.

@brondavies I tried updating RestSharp to 106.11 but it still breaks SeatsioDotNet

**Review by brondavies — APPROVED** [review]

**Pull request merged**